CONTRIBUTING.md

```diff
@@ -33,6 +35,41 @@ contributing.
    npm install
    ```
 
+#### Windows (using WSL2)
+
+1. Set up wsl and the linux distribution. WSL allows linux distributions to run on the Windows OS. Visit this [website](https://docs.microsoft.com/en-us/windows/wsl/install) for more information.
+
+   ```powershell
+   wsl --install
+   ```
+
+1. Follow instructions for [linux](#linux)
+
+#### Linux
+
+1. Install curl
+
+   ```sh
+   sudo apt-get install curl
```

Comment:
**question:** Interesting. Is it to support installing NVM? If the user opts to use `curl` to download NVM as per the [instruction](https://github.com/nvm-sh/nvm#install--update-script), I think this would be useful. Do you know if `wget` already comes with WSL or do we also need to install it manually as well?

Reply:
I think it depends on what distribution people download. Some have wget and curl preinstalled.

The one Morgaine and I installed did not  but I know there are versions that do.

This is more of a fail safe statement to prevent confusion.

Reply:
Thanks for clarifying @talzid. LGTM.